lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/ipv-core-main-journey.yaml

```diff
@@ -216,6 +216,18 @@ PYI_ESCAPE:
     end:
       targetState: END
 
+PYI_CRI_ESCAPE:
+  response:
+    type: page
+    pageId: pyi-cri-escape
+  events:
+    dcmaw:
+      targetState: CRI_DCMAW
+    f2f:
+      targetState: CRI_F2F_J4
```

Comment:
I'm thinking here we might want to define separate target states for the 2 options rather than pointing to existing journeys CRI_DCMAW and CRI_F2F_J4? Particularly for CRI_DCMAW all its fail states route to the doc check page which would be very confusing for the user/they could get stuck in a loop. I'm not sure product/UCD have considered this so will raise it with them